src/style.css

```diff
@@ -0,0 +1,88 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.block {
+  --height: 300px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+  flex-basis: var(--height);
+
+  height: var(--height);
+
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+.block:nth-child(1) {
+  background-color: rgb(255, 0, 0);
+  order: 1;
+}
+
+.block:nth-child(2) {
+  order: 2;
+  background-color: rgb(204, 0, 0);
+}
+
+.block:nth-child(3) {
+  order: 3;
+  background-color: rgb(153, 0, 0);
+}
+
+.block:nth-child(4) {
+  order: 4;
+  background-color: rgb(102, 0, 0);
+}
+
+.block:nth-child(5) {
+  order: 5;
+  background-color: rgb(51, 0, 0);
+}
+
+.block:nth-child(6) {
+  order: 6;
+  background-color: rgb(0, 0, 0);
+}
```

Comment:
you don't need these custom orders ere, just change them in your media rules

Reply:
Fixed